tables/rx_potency/chen21e-pot.csv

```diff
@@ -0,0 +1,118 @@
+ref_name,rx_name,iso_name,section,assay_name,potency_type,potency,cumulative_count,potency_upper_limit,potency_lower_limit,potency_unit,date_added
+Chen21e,COV2-2130,WA1+S:614G,Figure 1c,Virus isolate,IC50,26,1,10000,1,ng/ml,2022-01-08

```

Comment:
According to Source Data Fig. 1 workbook "Fig 1d", the potency_upper_limit is 10000 and the potency_lower_limit should be 0.128.

<img width="158" alt="Screen Shot 2022-01-23 at 12 43 35 PM" src="https://user-images.githubusercontent.com/809865/150697292-e50fdcc6-889b-40d3-9acb-3a6a04d365d6.png">